src/JuliaFormatter.jl

```diff
@@ -643,7 +643,7 @@ function format_file(
 )::Bool
     path, ext = splitext(filename)
     shebang_pattern = r"^#!\s*/.*\bjulia[0-9.-]*\b"
-    formatted_str = if ext == ".md" || ext == ".jmd"
+    formatted_str = if !isnothing(match(r"^\.[jq]*md$", ext))
```

Comment:
oops forgot about this initially but we can't use `isnothing` since it's not compatible with earlier Julia versions, need `===` or `!==` instead